### PR TITLE
add io.webtorrent.WebTorrent

### PR DESCRIPTION
**Pull request by prcastro**
Opened the app, downloaded a (legal) torrent, the file was downloaded to the correct place and a notification showed up. I was also able to play the torrent from the app.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build io.webtorrent.WebTorrent

File: io.webtorrent.WebTorrent.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        "chmod +x /app/bin/WebTorrent",
+        "install -Dm644 io.webtorrent.WebTorrent.desktop /app/share/applications/io.webtorrent.WebTorrent.desktop",
+        "install -Dm644 io.webtorrent.WebTorrent.appdata.xml /app/share/appdata/io.webtorrent.WebTorrent.appdata.xml"
+      ],
+      "cleanup": [
+        "/share/doc"
+      ],
+      "sources": [
+        {
+          "type": "file",
```
Comment (nedrichards) [inline io.webtorrent.WebTorrent.json]: Could you support the i386 version as well using [only arches like this](https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk9/blob/master/org.freedesktop.Sdk.Extension.openjdk9.json#L32). I see there's an i386 binary.

https://github.com/webtorrent/webtorrent-desktop/releases/download/v0.18.0/WebTorrent-v0.18.0-win-ia32.zip

**Comment by nedrichards** [comment]
This will need a flathub.json file ([example](https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk9/blob/master/flathub.json)) to prevent it building on arm or aarch64 since they don't distribute binaries for those architectures.

**Comment by prcastro** [comment]
Done

**Comment by nedrichards** [comment]
bot, build io.webtorrent.WebTorrent

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.webtorrent.WebTorrent

**Pull request closed without merging**